vllm/entrypoints/openai/serving_embedding.py

```diff
@@ -113,15 +133,6 @@ async def create_embedding(
 
         truncate_prompt_tokens = None
 
-        if request.truncate_prompt_tokens is not None:
```

Comment:
This verification was added to reproduce the behavior of the OpenAI embeddings API. It returns an error if the sequence length exceeds the model max length.

Reply:
Oops, didn't mean to remove this!